src/routes/index.ts

```diff
@@ -10,4 +11,8 @@ router.get('/api/subversions', (ctx) => {
   ctx.body = { '/api/media': mediaApiSubversions };
 });
 
+router.get('/api/image-prepend', async(ctx) => {
```

Comment:
I'm not sure what this endpoint is doing, but we may as well add the subversioning header to it, as per the other endpoints

Reply:
The TMDB API exposes this config endpoint to let them change their base URL, and then each image they return is a relative path, so to get an image you combine those two pieces of information. Cool I'll add the subversioning header

Reply:
@SubJunk perhaps the names of image prepend in the code could be referred to as image base url?